internal/backup/volumesnapshotcontent_action.go

```diff
@@ -70,12 +70,12 @@ func (p *VolumeSnapshotContentBackupItemActionV2) Execute(item runtime.Unstructu
 	if util.DataMoverCase() {
 
 		// check the VSC has the same backup name from label as the current backup
-		isVSForCurrentBackup := util.VSBHasVSBackupName(backup, &snapCont, p.Log)
+		isVSForCurrentBackup := util.VSCBelongsToBackup(backup, &snapCont, p.Log)
 
 		if !isVSForCurrentBackup {
-			p.Log.Warnf("stale volumesnapshot found %s", snapCont.Spec.VolumeSnapshotRef.Name)
+			p.Log.Infof("unrelated volumesnapshot found %s", snapCont.Spec.VolumeSnapshotRef.Name)
```

Comment:
```suggestion
			p.Log.Infof("unrelated volumesnapshotcontent found %s, skipping", snapCont.Spec.VolumeSnapshotRef.Name)
```

Reply:
@kaovilai I believe Scotts change is accurate here, its volumesnapshot and not volumesnapshotcontent

Reply:
This is under VSCItemAction

Line 72-76 for full context.
```
		// check the VSC has the same backup name from label as the current backup
		isVSForCurrentBackup := util.VSCBelongsToBackup(backup, &snapCont, p.Log)

		if !isVSForCurrentBackup {
			p.Log.Infof("unrelated volumesnapshot found %s", snapCont.Spec.VolumeSnapshotRef.Name)

```

Reply:
@shubham-pampattiwar @kaovilai so my change is accurate, but the "volumesnapshot" was there before. But yes, it's the VSC we're talking about here, as this is a VSC plugin, so changing the message as above fixes the pre-existing inaccuracy.